examples/speed_estimation/ultralytics_example.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, source: np.ndarray, target: np.ndarray) -> None:
         self.m = cv2.getPerspectiveTransform(source, target)
 
     def transform_points(self, points: np.ndarray) -> np.ndarray:
+        if points.size == 0:
+            return points
```

Comment:
AttributeError: 'NoneType' object has no attribute 'reshape'

Error was NoneType so ".size" can also cause same because NoneType doesn't have ".size" so you should check type instead of size first.

Reply:
That None is generated later. It is generated if point list is empty.